docs/contributing.md

```diff
@@ -70,7 +70,8 @@ The following steps are biased towards [vscode](https://code.visualstudio.com/):
             "type": "go",
             "request": "launch",
             "mode": "auto",
-            "program": "${workspaceFolder}"
+            "program": "${workspaceFolder}",
+            "args": [ "--zap-log-level=debug", "--zap-stacktrace-level=error" ]
```

Comment:
does it need to be added here: https://github.com/pulp/pulp-operator/blob/go-1.0.0-beta/.ci/scripts/local.sh#L13 ?

Reply:
I think that by setting `DEV_MODE` in zap will already define log-level to debug and stacktrace to warn
https://github.com/pulp/pulp-operator/blob/3130100f4570af853a4783be59f9f86693026850/.ci/scripts/local.sh#L17